### PR TITLE
Create ptp deployment manifests

### PR DESCRIPTION
**Pull request by SchSeba**
This PR add the ptp feature using kustomize.

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/openshift-kni/cnf-features-deploy/6)

### CODE REVIEW

File: feature-configs/demo/ptp/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
```
Comment (slintes) [inline feature-configs/demo/ptp/kustomization.yaml]: please use `resources`, `bases` is deprected

File: feature-configs/demo/ptp/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  - ../../base/ptp
+patches:
```
Comment (slintes) [inline feature-configs/demo/ptp/kustomization.yaml]: same here, please use `patchesStrategicMerge`

**Comment by SchSeba** [comment]
@slintes thanks for the comments can you please take another look?

@davidvossel we can't test this on the gcp e2e test we need a baremetal deployment. The ptp sync time depends on a physical network interface capabilities.

**Comment by davidvossel** [comment]
this is being blocked by the e2e tests. we don't actually want those to be required for merger. I've created a PR to resolve that, which should fix our situation here.

https://github.com/openshift/release/pull/6714

**Pull request merged**